paasta_tools/cli/cmds/status.py

```diff
@@ -838,13 +838,28 @@ def print_flink_status(
         f" {status.overview['slots-available']}/{status.overview['slots-total']} slots available"
     )
 
+    # Avoid cutting job name. As opposed to default hardcoded value of 32, we will use max length of job name
+    max_job_name_length = max(
+        [len(job["name"].split(".", 2)[2]) for job in status.jobs]
+    )
+    # Limiting longest allowed job name to be 120 chars. This is to ensure while printing on cli the formatting is maintained on normal screens
+    allowed_max_job_name_length = min(120, max_job_name_length)
```

Comment:
Please use the real width of the terminal, something like:
```
allowed_max_job_name_length = min(shutil.get_terminal_size((120, 50)).columns, max_job_name_length)
```
Also, please make sure that the whole line fits into the terminal and not only the job name column.

Reply:
that's actually a good idea +1

Reply:
Thanks, this makes more sense.